### ISSUE TITLE
ceteris_paribus: number of rows of the result

### ISSUE THREAD

**Issue by mstaniak** [ModelOriented/ingredients#13]
```
library(DALEX)
library(randomForest)
data('apartments')
m_rf_ap <- randomForest(m2.price ~., data = apartments[1:500, ], ntree = 10)
ap_explainer <- explain(m_rf_ap, apartments[1:500, ])

n_points <- min(101, length(unique(ap_explainer$data[, "construction.year"])))

ceteris <- ingredients::ceteris_paribus(
  ap_explainer, apartments[10, ], grid_points = n_points,
  variables = "construction.year")[, c("construction.year", "_yhat_", "_label_")]
nrow(ceteris)
n_points
```

Where is the difference coming from?

**Comment by mstaniak** [comment]
Also, notice that values of construction.year are slightly differ between data and ceteris_paribus.

**Comment by mstaniak** [comment]
Both things are related to this code:

```
probs <- seq(0, 1, length.out = grid_points)
unique(quantile(selected_column, probs = probs))
```

from the function that calculates variables splits. It causes two problems:
- it's harder to use ceteris_paribus in code for other functions, 
- some unique values of a feature disappear.

**Comment by pbiecek** [comment]
Sure, 
what would be a desired output?
All unique values in the variable?

One can create a `calculate_variable_split_unique` function that will be supplied as an argument for `ceteris_paribus`.

The original idea was to limit number of points in which we should evaluate CP profiles

**Comment by mstaniak** [comment]
Not necessarily all of them, but I think that if, say, a user specifies `grid_points = 100` and gets 90 without any message, it is a bit undesirable for programmatic purposes. I guess it's not an issue with standard applications of the package.

**Comment by pbiecek** [comment]
ok, would it be enough to change name of the parameter to `max_no_points` ?

**Comment by mstaniak** [comment]
A clear information about this behaviour is okay with me.

**Comment by pbiecek** [comment]
fixed in 100f5db
